tests/terraform/create_image.pm

```diff
@@ -0,0 +1,40 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Create HDD image that boots automatically and add a second
+#          interface config file.
+# Maintainer: Jose Lausuch <jalausuch@suse.com>
+
+
+use base 'opensusebasetest';
+use strict;
+use testapi;
+use warnings;
+
+sub run {
+    my ($self) = @_;
+
+    $self->select_serial_terminal;
+
+    # Don't wait for user to press enter, boot the system automatically
+    assert_script_run('sed -i s/GRUB_TIMEOUT=-1/GRUB_TIMEOUT=0/ /etc/default/grub');
+    assert_script_run('grub2-mkconfig -o /boot/grub2/grub.cfg');
+
+    # Allow having a second NIC configurable
+    assert_script_run('cp /etc/sysconfig/network/ifcfg-eth0 /etc/sysconfig/network/ifcfg-eth1');
+
+    # Allow any connection to the VM (e.g. ICMP, SSH, ...)
+    assert_script_run('systemctl disable firewalld');
+
+    # this is needed for the test 'sle15_workarounds' since it fails
+    # if the previous test has selected virtio console
```

Comment:
Maybe you can simply adjust sle15_workarounds to work with the serial terminal or just put the `select_console 'root-console'` there as this module should not depend on any other potential later module.

Reply:
yes, actually no need for serial console here